<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CleanForm</class>
 <widget class="QWidget" name="CleanForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>781</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QRoundProgressBar" name="roundProgressBar_DeviceCheck" native="true">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>250</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>开始一次</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_CardSender">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>431</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">      QGroupBox {
            border-color: rgb(156, 156, 156);
            border-width: 1px;
            border-style: solid;
            margin-top: 0.5ex;
			 background-color:grey;
        }
        QGroupBox::title {
            subcontrol-origin: margin;
            subcontrol-position: top left;
            left: 10px;
            margin-left: 2px;
            padding: 0  0px;
        }</string>
    </property>
    <property name="title">
     <string>发卡器</string>
    </property>
    <widget class="QPushButton" name="pushButton_CardSenderInit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>初始化</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CardSender">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>发卡</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CardSenderRequest">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查询传感器状态</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_SendMod">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>431</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">      QGroupBox {
            border-color: rgb(156, 156, 156);
            border-width: 1px;
            border-style: solid;
            margin-top: 0.5ex;
			 background-color:grey;
        }
        QGroupBox::title {
            subcontrol-origin: margin;
            subcontrol-position: top left;
            left: 10px;
            margin-left: 2px;
            padding: 0  0px;
        }</string>
    </property>
    <property name="title">
     <string>进卡机构</string>
    </property>
    <widget class="QPushButton" name="pushButton_SmInit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>初始化</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SmFlipin">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>转向进卡</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SmFlipout">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>转向出卡</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SmCleanIn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清洁进卡</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SmCleanOut">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清洁出卡</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SmCleanCache">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>缓冲出卡</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Sort">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>431</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">      QGroupBox {
            border-color: rgb(156, 156, 156);
            border-width: 1px;
            border-style: solid;
            margin-top: 0.5ex;
			 background-color:grey;
        }
        QGroupBox::title {
            subcontrol-origin: margin;
            subcontrol-position: top left;
            left: 10px;
            margin-left: 2px;
            padding: 0  0px;
        }</string>
    </property>
    <property name="title">
     <string>分拣模块</string>
    </property>
    <widget class="QPushButton" name="pushButton_Sort_LongOut">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>卡槽出卡</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SortModInit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>初始化</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_PrintMod2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">      QGroupBox {
            border-color: rgb(156, 156, 156);
            border-width: 1px;
            border-style: solid;
            margin-top: 0.5ex;
			 background-color:grey;
        }
        QGroupBox::title {
            subcontrol-origin: margin;
            subcontrol-position: top left;
            left: 10px;
            margin-left: 2px;
            padding: 0  0px;
        }</string>
    </property>
    <property name="title">
     <string>打印模块2</string>
    </property>
    <widget class="QPushButton" name="pushButton_PrintMod2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>初始化</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PM2_CLeanSlide">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清洁轨道</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PM2_S2Out">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>轨道2出卡</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_AllInit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>180</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>全部初始化</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>570</y>
      <width>440</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_PrintMod3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">      QGroupBox {
            border-color: rgb(156, 156, 156);
            border-width: 1px;
            border-style: solid;
            margin-top: 0.5ex;
			 background-color:grey;
        }
        QGroupBox::title {
            subcontrol-origin: margin;
            subcontrol-position: top left;
            left: 10px;
            margin-left: 2px;
            padding: 0  0px;
        }</string>
    </property>
    <property name="title">
     <string>打印模块3</string>
    </property>
    <widget class="QPushButton" name="pushButton_PrintMod3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>初始化</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PM3_CLeanSlide">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清洁轨道</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PM3_S2Out">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>轨道2出卡</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>460</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>431</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">      QGroupBox {
            border-color: rgb(156, 156, 156);
            border-width: 1px;
            border-style: solid;
            margin-top: 0.5ex;
			 background-color:grey;
        }
        QGroupBox::title {
            subcontrol-origin: margin;
            subcontrol-position: top left;
            left: 10px;
            margin-left: 2px;
            padding: 0  0px;
        }</string>
    </property>
    <property name="title">
     <string>OCR设备</string>
    </property>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>初始化</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清洁</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Pause">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>390</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_PrintMod1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">      QGroupBox {
            border-color: rgb(156, 156, 156);
            border-width: 1px;
            border-style: solid;
            margin-top: 0.5ex;
			 background-color:grey;
        }
        QGroupBox::title {
            subcontrol-origin: margin;
            subcontrol-position: top left;
            left: 10px;
            margin-left: 2px;
            padding: 0  0px;
        }</string>
    </property>
    <property name="title">
     <string>打印模块1</string>
    </property>
    <widget class="QPushButton" name="pushButton_PrintMod1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>初始化</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PM1_CLeanSlide">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清洁轨道</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PM1_S2Out">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>轨道2出卡</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Lam">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">      QGroupBox {
            border-color: rgb(156, 156, 156);
            border-width: 1px;
            border-style: solid;
            margin-top: 0.5ex;
			 background-color:grey;
        }
        QGroupBox::title {
            subcontrol-origin: margin;
            subcontrol-position: top left;
            left: 10px;
            margin-left: 2px;
            padding: 0  0px;
        }</string>
    </property>
    <property name="title">
     <string>覆膜模块</string>
    </property>
    <widget class="QPushButton" name="pushButton_LamModInit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>初始化</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_LamCleanSlide">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清洁轨道</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_LamOut">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>轨道2出卡</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit_Send">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>440</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>553</y>
      <width>30</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>553</y>
      <width>41</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Receive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StartWhile">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>320</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>循环开始</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_SEND">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_P1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_P2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_P3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>380</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_LAM">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>440</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_SORT">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>500</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_1">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>260</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>启用</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>320</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>启用</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>380</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>启用</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRoundProgressBar</class>
   <extends>QWidget</extends>
   <header>QRoundProgressBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
